StripeFinancialConnections/StripeFinancialConnections/Source/Helpers/Locale+Extensions.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Locale+Extensions.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 6/20/23.
+//
+
+import Foundation
+
+extension Locale {
+
+    /// Returns the BCP 47(-ish) language tag representing the locale.
+    ///
+    /// The language tag is expected to be well-formed as log as the locale identifier contains a
+    /// valid language code. For example:
+    ///
+    /// ```
+    /// let locale = Locale(identifier: "fr_CA")
+    /// locale.toLanguageTag() // -> "fr-CA"
+    /// ```
+    ///
+    /// The following example returns `"-ES"`, even though `"und-ES"` will be the appropriate BCP 47 tag:
+    ///
+    /// ```
+    /// let locale = Locale(identifier: "_ES")
+    /// locale.toLanguageTag() // -> "-ES"
+    /// ```
+    ///
+    /// All system iOS and macOS locales are expected to contain valid language codes.
+    func toLanguageTag() -> String {
```

Comment:
it seems like regardless of whatever changes in this PR, this seems like a _more_ expected format than the previous `Locale.current.identifier`

Reply:
this comes from here:
- https://github.com/stripe/stripe-ios/blob/52e9b64c/StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Utils/Locale+Link.swift#L31